.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: ci
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}-latest
```

Comment:
Let's run on both Ubuntu 18.04 and 20.04 explicitly.

Reply:
Done